Src/IronPython/IronPython.csproj

```diff
@@ -1,511 +1,85 @@
+    </None>
+    <None Update="Lib\iptest\file_util.py">
       <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
-    </Content>
-    <Content Include="Lib\iptest\file_util.py">
+    </None>
+    <None Update="Lib\iptest\misc_util.py">
       <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
-    </Content>
-    <Content Include="Lib\iptest\misc_util.py">
+    </None>
+    <None Update="Lib\iptest\process_util.py">
       <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
-    </Content>
-    <Content Include="Lib\iptest\process_util.py">
+    </None>
+    <None Update="Lib\iptest\runner.py">
       <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
-    </Content>
-    <Content Include="Lib\iptest\runner.py">
+    </None>
+    <None Update="Lib\iptest\test_env.py">
       <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
-    </Content>
-    <Content Include="Lib\iptest\test_env.py">
+    </None>
+    <None Update="Lib\iptest\type_util.py">
       <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
-    </Content>
-    <Content Include="Lib\iptest\type_util.py">
+    </None>
+    <None Update="Lib\iptest\util.py">
       <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
-    </Content>
-    <Content Include="Lib\iptest\util.py">
+    </None>
+    <None Update="Lib\iptest\warning_util.py">
       <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
-    </Content>
-    <Content Include="Lib\iptest\warning_util.py">
+    </None>
+    <None Update="Lib\iptest\__init__.py">
       <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
-    </Content>
-    <Content Include="Lib\iptest\__init__.py">
+    </None>
+    <None Update="Lib\runpy.py">
       <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
-    </Content>
-    <Content Include="Lib\runpy.py">
+    </None>
+    <None Update="Lib\site.py">
       <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
-    </Content>
-    <Content Include="Lib\__future__.py">
+    </None>
+    <None Update="Lib\__future__.py">
       <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
-    </Content>
+    </None>
   </ItemGroup>
+   <Import Project="Sdk.targets" Sdk="Microsoft.NET.Sdk" />
+  <Import Project="$(BeforeTargetFiles)" />
+  <Target Name="BeforeBuild" DependsOnTargets="$(BeforeTargets);GenerateCurrentVersion" />
   <ItemGroup>
-    <EmbeddedResource Include="Resources.resx">
-      <SubType>Designer</SubType>
-      <Generator>ResXFileCodeGenerator</Generator>
-      <LastGenOutput>Resources.Designer.cs</LastGenOutput>
-    </EmbeddedResource>
+    <!-- if the file does not exist it's not picked up automatically -->
+    <Compile Remove="Properties\CurrentVersion.Generated.cs" />
+    <Compile Include="Properties\CurrentVersion.Generated.cs" />
+    <Compile Remove="Properties\BuildInfo.Generated.cs" />
+    <None Remove="Modules\unicodedata\IPyUnicodeData.txt.gz" />
```

Comment:
Do we need the remove here? If so maybe move it to next to the include of same file.